lib/normalizePath.js

```diff
@@ -1,10 +1,12 @@
 import escapeRegExp from 'lodash.escaperegexp';
 
+const CWD_PATTERN = RegExp(`^${escapeRegExp(process.cwd())}`);
```

Comment:
Yep. I was afraid that it would lock us down to not being able to change cwd mid-process. But after looking at this more, we can't really switch cwd anyway. Mostly because of the `fix-node-path` thing, which can only happen on initialization.